### PR TITLE
W-11020951-mfa

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/multi-factor-authentication.adoc [hunk 1]
```diff
@@ -112,6 +114,7 @@ The *Manage Your Verification Methods* interface appears.
 If your organization requires multi-factor authentication and you have removed all verification methods, you must configure at least one verification method before you save your changes.
 . Click *Save*.
 
+[[disable-org-mfa]]
 == Disable Multi-Factor Authentication Requirement for All Organization Users
 
 When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.
```
Comment (ductran-ms) [inline modules/ROOT/pages/multi-factor-authentication.adoc]: I think it is outdated. You cannot disable MFA. You can make it either optional or mandatory

File: modules/ROOT/pages/multi-factor-authentication.adoc [hunk 2]
```diff
@@ -21,6 +21,7 @@ After multi-factor authentication is enabled, users must verify their Anypoint P
 * Security key
 * Salesforce Authenticator
 
+Organizations created after April 30, 2022 require multi-factor authentication by default for all users. Organization administrators can xref:multi-factor-authentication.adoc#exempt-mfa-users[add individual user and service accounts to an exemption list] or xref:multi-factor-authentication.adoc#disable-org-mfa[disable MFA across the organization] (not recommended). Note that https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce users must have MFA enabled].
```
Comment (ductran-ms) [inline modules/ROOT/pages/multi-factor-authentication.adoc]: I don't this option "disable MFA across the organization".
I'm not sure if "Salesforce users must have MFA enabled" is relevant to anypoint.

Comment (sarathecoubian) [inline modules/ROOT/pages/multi-factor-authentication.adoc]:
```suggestion
Organizations created after April 30, 2022 require multi-factor authentication by default for all users. Organization administrators can xref:multi-factor-authentication.adoc#exempt-mfa-users[add individual user and service accounts to an exemption list]. Note that https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce users must have MFA enabled].
```

File: modules/ROOT/pages/multi-factor-authentication.adoc (hunk 1 above)
Comment (sarathecoubian) [inline modules/ROOT/pages/multi-factor-authentication.adoc]:
```suggestion
NOTE:
Organizations created after April 30, 2022 cannot disable the multi-factor authentication requirement in their organizations and must add individual accounts to the exemption list to bypass the requirement.

When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.
```

File: modules/ROOT/pages/multi-factor-authentication.adoc [hunk 3]
```diff
@@ -112,8 +114,12 @@ The *Manage Your Verification Methods* interface appears.
 If your organization requires multi-factor authentication and you have removed all verification methods, you must configure at least one verification method before you save your changes.
 . Click *Save*.
 
+[[disable-org-mfa]]
 == Disable Multi-Factor Authentication Requirement for All Organization Users
 
+[NOTE]
+Organizations created after April 30, 2022 cannot disable the multi-factor authentication requirement in their organizations and must add individual accounts to the exemption list to bypass the requirement.
+
 When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/multi-factor-authentication.adoc]:
```suggestion
When you disable the multi-factor authentication (MFA) requirement, new users who log in to the organization do not receive a prompt to configure MFA. Note that disabling MFA does not automatically remove MFA from user accounts. Individual users who have configured multi-factor authentication must manually remove it in their profile settings.
```

**Review by ductran-ms — APPROVED** [review]

**Review by fermujica — APPROVED** [review]

**Pull request merged**